flokk_src/lib/_internal/universal_file/io_file.dart

```diff
@@ -7,7 +6,8 @@ import 'package:path/path.dart' as p;
 import 'universal_file.dart';
 
 class IoFileWriter implements UniversalFile {
-  Directory dataPath;
+  late Directory dataPath;
```

Comment:
Not sure how safe this is??? Probably worth having this actually treated as nullable?